src/components/YouAndYourFamilyCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ParticipantInfo from './ParticipantInfo';
+import participantData from './data/participantData.js';
+import '../index.css';
+import OverviewCard from './OverviewCard';
+
+/**
+ * Issues
+ * - cannot get 'add' btn to show inside the OverviewCard
+ * - cannot get 'add' to open the popup (somethings wrong with communication between children and parents)
+ */
+
+function YouAndYourFamilyCard(props) {
+    const participantComponents = participantData.map(data => <ParticipantInfo key={data.id} name={data.name} age={data.age} dutypreferences={data.dutypreferences} carseat={data.carseat}/>);
+    const CardHeader = ["Name","Age","Duties","Carseat"];
+    const Rightbuttons = ["Add"];
+    const LeftButtons = ["Delete"];
+
+    return (
+        <div className="YouAndYourFamily">
+            <OverviewCard active={props.active} class="YouAndYourFamily" header={CardHeader} content={participantComponents} rightbutton={Rightbuttons} leftbutton={LeftButtons}/>
```

Comment:
Bit much on one line. Don't be afraid to use more lines for element properties and component props